e2e/integration/menu.e2e.ts

```diff
@@ -85,7 +93,7 @@ describe('Menu', () => {
     cy.url().should('include', 'deletePetBy%22Id');
   });
 
-  it.only('should encode URL when the operation IDs have backslashes', () => {
```

Comment:
I'm assuming this `only` was a leftover from the PR that introduced it. Removed to allow all tests in the file to run